tests/jupyterhub_api/test_stop_session.py

```diff
@@ -0,0 +1,110 @@
+"""
+
+Test to stop the session
+
+To run the test use the following command:
+python3 test_stop_session.py --port 443 --users user1
+
+For multiple users
+
+python3 test_stop_session.py --port 443 --users user0 user1 user2
+
+"""
+import json
+import requests
+import subprocess
+import os
+import time
+import argparse
+import sys
+sys.path.append("..")
+import time
+from logger import Logger, LOG_FOLDER, LOG_EXTENSION
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+import yaml
+
+
+def get_args():
+    parser = argparse.ArgumentParser(description='Arguments', formatter_class = argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument("--port", dest="port", type=int,
+                        required = True,
+                        )
+    parser.add_argument( "--users", nargs='+', dest="users",
+                        required  = True,
+                        help='list of users')
+
+    args = parser.parse_args()
+    return args
+
+class StopSession:
+    def __init__(self, port, users, verify):
+        self.port = port
+        self.users = users
+        self.main_url = "https://localhost:" + str(self.port) + "/hub/api/"
+        self.ref_test_name ="Stop_Session"
+        self.exit = 0
+        self.verify = verify
+        self.token = ""
+        self.ref_timestamp = int(time.time())
+        self.logger_folder = os.path.join(os.getcwd(), LOG_FOLDER)
+        self.log = Logger(os.path.join(self.logger_folder,
+                                       self.ref_test_name + "_" + time.strftime("%Y-%m-%d_%H:%M:%S") + LOG_EXTENSION))
+        self.log_params()
+        self.get_tokens()
+
+    def log_params(self):
+        self.log.write("parameters", "Test name: " + self.ref_test_name)
+        self.log.write("parameters", "Test time: " + str(self.ref_timestamp))
+        self.log.write("parameters", "Logger folder: " + self.logger_folder)
+
+
+    def get_tokens(self):
```

Comment:
Please create a unique function to load the token that is called by all the tests: test_stop_session, test_create_session, test_check_session